### DISCUSSION TITLE
Where is my guests' "disks"? And more/better tutorials?

### DISCUSSION THREAD

**Discussion by UdderNonsense** [utmapp/UTM#5700] [General]
I recently discovered UTM after deciding I wanted to have a virtual machine for my Linux course.

Right now, I am trying to use Fedora Workstation 38 through its live ISO. Everything works by just clicking through and following instructions, but in the guest I am prompted to install workstation 38 (as if I have booted from a flash drive). These things make sense to me more or less, until after I install it and restart Fedora.

It is clear that UTM would boot from the live ISO it was configured for, but I don't know how to configure it for the proper Fedora installation: I don't even know where the installation went.

I've noticed that when attempting the install a second time, the disk space I gave to the guest (through UTM) is suddenly full. An almost entirely free 64 GB disk drops to 2 MB free, according to the installer. This goes away when I delete the guest through UTM and start the whole thing over.

tl;dr where are the guests' "disks" located on the host machine, 

Also, are there any video tutorials or step by step guides for precisely what I am doing? The docs I've seen on UTM's website have not clarified much. To be fair, I haven't looked much deeper than what the sections titles imply.
